packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,344 @@
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/font
+  // Heading tags inherit all font style rules to match body text;
+  // This ensures semantics are distinct from appearance.
+  font: inherit;
+  // margin applied to top of heading, informed by current margin size
+  margin-block: 1.5em 0;
+}
+
+p {
+  // ❓margin applied to top of text, informed by Chrome UA style - do we want to codify it?
+  margin-block: 1em 0;
+
+  // ❓taken from ./_base.scss ln 329 - do we want to add this to the reset?
+  // CMS-WDS/application @extends this style for their reset:
+  // https://github.cms.gov/CMS-WDS/application/blob/6de8f3ac19e86b7653f0102dcfff7a79241dcb33/app/styles/base/index.scss#L14
+  &:first-child {
+    margin-block-start: 0;
+  }
+}
+
+// These tags will naturally use 700 weight as bold
+// but because some themes have 600 weight, I'm using
+// the CSS variable to adjust bold based on theme
+b,
+strong {
+  font-weight: var(--font-weight-bold);
+}
+
+a {
+  color: var(--link__color);
+  // 📢 We may want to revisit tokens for `text-decoration-style`
+  // ❓Do we have an instance where the underline doesn't match the text color?
+  text-decoration-color: currentColor;
+  text-decoration-style: solid;
+  text-decoration-thickness: var(--link__text-decoration-thickness);
+  text-underline-offset: var(--link__text-decoration-offset);
+
+  // Recommended order of user action pseudo-classes:
+  // https://css-tricks.com/snippets/css/link-pseudo-classes-in-order/
+
+  &:visited {
+    color: var(--link__color--visited);
+  }
+
+  &:hover {
+    color: var(--link__color--hover);
+    text-decoration-thickness: var(--link__text-decoration-thickness--hover);
+  }
+
+  &:focus,
+  &:focus:visited {
+    background-color: var(--color-focus-light);
+    box-shadow: none;
+    color: currentColor;
+    outline: 3px solid var(--color-focus-dark);
+    outline-offset: 1px;
+  }
+
+  &:active {
+    color: var(--link__color--active);
+  }
+}
+
+// ❓ How do we want to style <hr> tags? I'm using best guess here.
+hr {
+  border-block-end: 0;
+  border-block-start: 1px solid currentColor;
+  margin: 0;
+}
+
+/* Lists */
+
+// Relying on `flex` to set `gap` spacing size on li elements
+// `gap` ensures even spacing and doesn't interfere with layout if list given a `row` direction
+ol,
+ul {
+  // display: flex;
+  // flex-direction: column;
+  // gap: .5em;
+  margin-block: 1em 0;
+  padding-inline-start: 2em;
+
+  // Tying unstyled list styles to a required a11y attribute ensures proper markup structure
+  &[role='list'] {
+    list-style: none;
+    padding: 0;
+  }
+}
+
+// 📢 We don't have examples of nested lists in the design system
+// ❓ How do we handle li `::mark` and nested padding for this pattern?
+// If we choose to support nesting lists, need to remove `gap` solution in favor of li `margin-block-start`
+// li elements are `display: list-item` and cannot receive `flex` or `grid` display without removing `::mark`
+li {
+  margin-block: 0.5em 0;
```

Comment:
I eventually went back to the `flex` solution. The original problem I had with it was when nesting lists, there was inconsistent spacing between the `li` and `ul`/`ol` nested within it. That's because `li` has a `display: list-item` rule that can't really be changed to flex without losing the bullets or numbers (important for a list!).

I found a way to target `ul`/`ol` children of `li` els so they could be given consistent padding, meaning the flex solution now works perfectly.

I like the `flex` solution because it results in cleaner UI when teams render lists as rows or inline. Currently, our `li` els have this margin-top baked in, so when users create inline lists, there's this weird top spacing they have to contend with. Now that we have this gap solution, the spacing goes away. Teams can also create inline lists by styling them to `flex-direction: row` or adjust spacing between `li` with `gap: <WHATEVER UNIT>`.